docs/apv.md

```diff
@@ -538,6 +533,12 @@ Onderstaande regels tellen alleen op Narcopolis
 1. Het campen van een niet actieve drugslocatie Narcopolis is niet toegestaan.
 2. Het overtreden van deze regel zal resulteren in een straf uit de 2e categorie.
 
+### Artikel N-5 - Samenwerking criminele organisaties
+
+1. Op Narcopolis is het voor een officiële en onofficiele groep niet toegestaan om samen te werken met een officiele of onofficiele groep of persoon.
```

Comment:
Hier mee wordt uitgesloten dat er drugs wordt gekocht van personen op het eiland? aangezien dit samen werken is technisch gezien

Reply:
Ik heb er nog een lid aan toegevoegd, zodat het wel kan